tests/Message/MessageTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Message;
+
+use PhpLlm\LlmChain\Message\Message;
+use PhpLlm\LlmChain\Response\ToolCall;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\Attributes\Small;
+use PHPUnit\Framework\Attributes\UsesClass;
+use PHPUnit\Framework\TestCase;
+
+#[CoversClass(Message::class)]
+#[UsesClass(ToolCall::class)]
+#[Small]
+final class MessageTest extends TestCase
+{
+    public function testCreateSystemMessage(): void
```

Comment:
We could use Test Attribute instead of the prefix in the method name. Interested? If yes I can provide a PR

Reply:
Interesting, never ever used that. What's pro or con? Or rather style?

Reply:
Rather style, as I switched from `@test` annotation to `#[Test]` attribute